lua/lspsaga/callhierarchy.lua

```diff
@@ -245,17 +245,23 @@ function ch:keymap()
         return
       end
       local data = self.method == get_method(2) and curnode.value.from or curnode.value.to
-      local fname = vim.uri_to_fname(data.uri)
       local start = data.selectionRange.start
-      local pos = { start.line + 1 }
       self:clean()
       local restore = win:minimal_restore()
-      vim.cmd[action](fname)
+      vim.cmd(action)
```

Comment:
could we do action with file name ? not action then use jump_to_location to open file if that function failed the file not open right ?

Reply:
fair enough .